src/base-rest-resource.ts

```diff
@@ -146,8 +146,8 @@ class Base {
       if (error instanceof HttpResponseError) {
         throw new RestResourceRequestError({
           message: error.message,
-          code: error.code,
-          statusText: error.statusText,
+          code: error.response.code,
+          statusText: error.response.statusText,
```

Comment:
we don't include the body, headers or statustext for REST requests?

Reply:
Good catch, they should definitely be included here.